evm/verify-contract.js

```diff
@@ -320,6 +330,46 @@ async function processCommand(config, chain, options) {
             break;
         }
 
+        case 'AxelarAmplifierGateway': {
+            let args = options.args;
+
+            if (args) {
+                args = JSON.parse(args);
+                options = { ...options, ...args };
+            }
+
+            const AxelarAmplifierGateway = require('@axelar-network/axelar-gmp-sdk-solidity/artifacts/contracts/gateway/AxelarAmplifierGateway.sol/AxelarAmplifierGateway.json');
+            const amplifierGatewayFactory = await getContractFactoryFromArtifact(AxelarAmplifierGateway, wallet);
+            const amplifierGateway = amplifierGatewayFactory.attach(options.address || chain.contracts.AxelarGateway.address);
+
+            const implementation = await amplifierGateway.implementation();
```

Comment:
see latest amplifier deployment script / devnet-amplifier config. `proxyDeploymentArgs` are checked in now, switch to using that. The queried value can't be used for proxy verification (but only for the implementation verification)